src/controller.cpp

```diff
@@ -105,13 +105,13 @@ bool Ctrl::runCalculation(){
   if (data.success){
     notifyUser("Result for " + data.chemical_formula);
     notifyUser("Elapsed time: " + std::to_string(data.getTime()) + " s");
-    notifyUser("Van der Waals volume: " + std::to_string(data.volumes[0b00000011]) + " A^3");
-    notifyUser("Excluded void volume: " + std::to_string(data.volumes[0b00000101]) + " A^3");
-    notifyUser("Probe 1 core volume: " + std::to_string(data.volumes[0b00001001]) + " A^3");
-    notifyUser("Probe 1 shell volume: " + std::to_string(data.volumes[0b00010001]) + " A^3");
+    notifyUser("Van der Waals volume: " + std::to_string(data.volumes[0b00000011]) + " �^3");
+    notifyUser("Excluded void volume: " + std::to_string(data.volumes[0b00000101]) + " �^3");
+    notifyUser("Probe 1 core volume: " + std::to_string(data.volumes[0b00001001]) + " �^3");
+    notifyUser("Probe 1 shell volume: " + std::to_string(data.volumes[0b00010001]) + " �^3");

```

Comment:
None of these lines are output on macOS. I assume it's because of the Å

Reply:
Interesting. Does the report file output properly?

Reply:
Yes that works fine